content/topics/android/image-assets/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Working with image assets
+ready: true
+---
+
+In Android images can be uploaded into the drawables folder that will be packaged along with the apk when it is compiled. These  images are uploaded into your project 
```

Comment:
I'd word it a little differently. A lot of our recruits aren't first-language english speakers. And when explaining things it's often best to have very short, very precise sentences.  Something like:

In Android, images can be added to a directory named `drawables`. The `drawables` directory will be packaged along with the apk when it is compiled. Your Android application can access this directory.